Week 1

Arrisio opened the pull request:
Реализовано согласно требованиям 
![Loft-Taxi-SignIn](https://user-images.githubusercontent.com/14781165/78420782-14208b00-765b-11ea-935a-20ea0296afe3.png)
![Loft-Taxi-SignUp](https://user-images.githubusercontent.com/14781165/78420783-1551b800-765b-11ea-8700-eb7c752aca6a.png)
![Loft-Taxi-MapPage](https://user-images.githubusercontent.com/14781165/78420784-184ca880-765b-11ea-88cb-01eadb1bc9df.png)

Commit:
comlete layout draft

Commit:
refactor: optimize code with replace classes from functioin arguments to css

Arrisio committed:
feat(navigation): goTo SignUp , gotTo mapPage

Arrisio committed:
fix(navigation): fix link behavior: goto signIn instead of mapPage

Arrisio committed:
style: pretify code

bread-bread commented on src/components/intro-page/index.js:
```diff
@@ -0,0 +1,49 @@
+    };
+
+    gotoSignUp = () => {
+        this.setState(() => {
```
Здесь запись можно сократить так: this.setState({ currentForm: 'sign-in-form' });
функцию в setState в этом случае можно не передавать, так как не нужно использовать актуальное состояние

bread-bread commented on src/components/intro-page/index.js:
```diff
@@ -0,0 +1,49 @@
+    };
+
+    gotoSignIn = () => {
+        this.setState(() => {
```
здесь так же

bread-bread commented on src/components/map-page/index.js:
```diff
@@ -0,0 +1,15 @@
+const MapPage = () => (
+    <div>
+        <Header/>
+        <div><Map/></div>
```
в div здесь не обязательно оборачивать

bread-bread commented on src/components/registation-form/index.js:
```diff
@@ -0,0 +1,13 @@
+
+import RegistrationForm from "../sign-in-form";
+
+const App = () => {
```
это здесь наверно лишнее?

bread-bread commented on src/components/sign-up-form/index.js:
```diff
@@ -0,0 +1,54 @@
+import ButtonActive from "../common/button-active";
+import InputText from "../common/input-text";
+
+const SignupForm = ({gotoSignInHandler, signInHandler}) => {
```
для стрелочных функций есть короткий вариант написания без return.
`const SignupForm = ({ arg }) => (<form>{some code}</form>)`

bread-bread commented on src/components/header/index.js:
```diff
@@ -0,0 +1,32 @@
+        {name: 'Карта', value: 'map', id: 1},
+        {name: 'Профиль', value: 'profile', id: 2},
+        {name: 'Выйти', value: 'login', id: 3}
+    ];
```
перед return хорошо бы иметь пробел

bread-bread commented on src/components/intro-page/index.js:
```diff
@@ -0,0 +1,49 @@
+                    </div>
+                    <div className="intro-page--form-wrapper">
+                        {
+                            this.state.currentForm === "sign-in-form" ?
```
хорошо бы доставать значения из стейт с помощью деструктуризации, в коде будет лаконичнее.
`const { currentForm } = this.state;`
и потом в коде просто:
`currentForm === 'something'...`

Arrisio committed:
refactor: removed unneccassery block

https://github.com/Arrisio/loft-taxi/pull/1#discussion_r403446091

Arrisio committed:
refactor: removed unwanted files

https://github.com/Arrisio/loft-taxi/pull/1#discussion_r403446091

Arrisio committed:
refactor: removed unwanted files

https://github.com/Arrisio/loft-taxi/pull/1#discussion_r403446091

Arrisio committed:
feat(navigation): add navigataion to menu in header

"по функционалу нужно сделать чтобы по клику на пункт меню обрисовывался
соответствующий ui, сейчас идёт перезагрузка страницы"

Arrisio committed:
feat(navigation): add navigataion to menu in header

"по функционалу нужно сделать чтобы по клику на пункт меню обрисовывался
соответствующий ui, сейчас идёт перезагрузка страницы"

Arrisio committed:
style: minimize arrow functions

Arrisio committed:
style: minimize arrow functions

Arrisio committed:
refactor: minimize setState

https://github.com/Arrisio/loft-taxi/pull/1#discussion_r403445803

Arrisio committed:
refactor: minor imporvement
